vertical-pod-autoscaler/pkg/decode-histogram/main.go

```diff
@@ -50,6 +51,23 @@ func main() {
 	aggregateState := model.NewAggregateContainerState()
 	// Load data into the AggregateContainerState using LoadFromCheckpoint
 	err = aggregateState.LoadFromCheckpoint(&checkpoint.Status)
+	if checkpoint.Namespace == "vpa-test-service" {
```

Comment:
Cannot move this logic into the LoadFromCheckpoint function (which I think would be better since this logic is all related to loading checkpoints) since the LoadFromCheckpoint function takes the CheckpointStatus specifically as the argument (unable to access annotations in the function).